language/English/strings.po

```diff
@@ -11579,7 +11579,43 @@ msgctxt "#21470"
 msgid "%s [%d]"
 msgstr ""
 
-#empty strings from id 21471 to 21601
+#: system/settings/settings.xml
+msgctxt "#21471"
+msgid "Initially"
+msgstr ""
```

Comment:
This for "Select the first unwatched TV show season/episode" - "Never/Initially/Always" right?

I don't understand what "Initially" does in this context, can you explain setting Initially does?